### PR TITLE
Format and check shell scripts

### PR DESCRIPTION
**Pull request by jooola**
Another large formatting / linting PR, that should have been done a long time ago.
Sorry to rush it right before we wanted our bug fixing "party" tomorrow.

I cannot set this up as pre-commit hook, because everyone would have to install shfmt and shellcheck on there system, so I'd rather put a CI check and let people deal with it.

I also added a project wide Makefile to run those checking/formatting commands. Will be useful in the future.

I'd like to embrace the following guidelines: https://google.github.io/styleguide/shellguide.html

Missing:
- [x] CI check
- [x] Fix Shellcheck warnings

### CODE REVIEW

File: install [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
 
 if [[ $EUID -ne 0 ]]; then
-    echo "Please run as root user."
-    exit 1
+  echo "Please run as root user."
+  exit 1
 fi
 
-SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+SCRIPT_DIR="$( cd "$( dirname "$0")"  && pwd)"
```
Comment (paddatrapper) [inline install:11]: Inconsistent spacing here, the ` ` after the opening bracket should be removed too

File: airtime_mvc/tests/runtests.sh [hunk 2]
```diff
@@ -29,9 +29,8 @@ export RABBITMQ_VHOST
 export AIRTIME_UNIT_TEST="1"
 
 #Change the working directory to this script's directory
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}")"  && pwd)"
```
Comment (paddatrapper) [inline airtime_mvc/tests/runtests.sh:32]: Inconsistent spacing here

**Review by paddatrapper — COMMENTED** [review]
Looked through some of it

**Comment by jooola** [comment]
Soo, this PR needs #1295 to be merged first.

Otherwise, I am gonna take a look at the diff, see any weird changes.

**Comment by paddatrapper** [comment]
I wonder how many of the legacy scripts we can just remove... Leaving this here to remind myself to go through the list

**Comment by jooola** [comment]
> I wonder how many of the legacy scripts we can just remove... Leaving this here to remind myself to go through the list

Just say the word ! I'll remove some of them with pleasure.

But just like airtime_mvc, all of this can be moved in a `legacy` folder to keep everything clean and ignored from our test tools. If someone needs it, it still can be integrated in the modern tools we are gonna build.

Or maybe an archive repository ?

**Comment by jooola** [comment]
Ok, I think this is good to go, just need to rebase on master once #1295.

**Comment by jooola** [comment]
Rebased, and ready to be merged.

**Comment by paddatrapper** [comment]
> Just say the word ! I'll remove some of them with pleasure.

Let's leave them for the moment and can sort through them in a different PR

**Comment by paddatrapper** [comment]
@jooola can you please add a description of the makefile and dependencies for checking shell scripts to a page under https://github.com/LibreTime/libretime/wiki#testing-changes?

**Pull request merged**